Description: include xfifo.h to avoid implicit pointer conversion
Author: Logan Rosen <logan@ubuntu.com>
Forwarded: no
Last-Update: 2016-02-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/filter.c
+++ b/src/filter.c
@@ -45,6 +45,7 @@
 #include "libprtX.h"
 #include "optBase.h"
 #include "linux_cmn.h"
+#include "xfifo.h"
 
 #define WIDTH_BYTES(bits) (((bits) + 31) / 32 * 4)
 
